bundles/org.openhab.binding.argoclima/README.md

```diff
@@ -0,0 +1,423 @@
+| schedule2OffTime       | text            | The time of day (HH:MM) the device should turn **OFF** on the ```schedule2DayOfWeek```-specified days. In effect only if ```activeTimer``` [channel](#channels) is in ```SCHEDULE_TIMER_2``` mode                                                              | 20:00                                | no       | yes      |
+| schedule3DayOfWeek     | text(multiple)  | Days (set comprising of values  ```MON```, ```TUE```, ```WED```, ```THU```, ```FRI```, ```SAT```, ``SUN``), when Schedule Timer 1 actions should be performed. This is used only if ```activeTimer``` [channel](#channels) is in ```SCHEDULE_TIMER_1``` mode.  | [SAT, SUN]                           | no       | yes      |
+| schedule3OnTime        | text            | The time of day (HH:MM) the device should turn **ON** *(in the last used mode)* on the ```schedule3DayOfWeek```-specified days. In effect only if ```activeTimer``` [channel](#channels) is in ```SCHEDULE_TIMER_3``` mode                                     | 11:00                                | no       | yes      |
+| schedule3OffTime       | text            | The time of day (HH:MM) the device should turn **OFF** on the ```schedule3DayOfWeek```-specified days. In effect only if ```activeTimer``` [channel](#channels) is in ```SCHEDULE_TIMER_3``` mode                                                              | 22:00                                | no       | yes      |
+| resetToFactoryDefaults | boolean(action) | When set, upon successful Thing initialization, the binding will issue a one-time factory reset request to the device (and flip this value back do OFF)                                                                                                        | false                                | no       | yes      |
+
+## Channels
+
+Both thing types are functionally equivalent and support the same channels.
+
+| Channel                | Type                 | Read/Write | Description                                                                                                                                                                                                            |
+|------------------------|----------------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| - **A/C Controls**                                                                                                                                                                                                                                                               ||||
```

Comment:
Can you fix the table markdown?

Reply:
If this is only about the "group header" rows, I'd prefer to keep them as written, if you don't mind...

While [GitHub-flavored Markdown](https://github.github.com/gfm/) doesn't support `colspan` _(and I definitely don't want to convert it into HTML)_, I've consciounsly opted for writing it this way (using [MultiMarkdown](https://github.com/fletcher/MultiMarkdown/wiki/MultiMarkdown-Syntax-Guide#tables) syntax), to signify it is meant as a  sub-header of sorts.

When viewed in a text editor it shows more akin to the display I wanted, ex. this MultiMarkdown version, with actual colspan:
![image](https://github.com/openhab/openhab-addons/assets/2566147/2673a08b-b709-4fbb-93f7-6633baa89e79)

That said, GFM doesn't support it (at least as for now) and it **is** single-cell-rendered indeed, so you're correct about the formatting technically being _wrong_. Since the last "sub-header" is rather long, I slightly prefer not to push all the columns by this offset as it'll make it actually worse to read in a text editor.

----

Keeping as is for now. If you still want it changed, considering explanation above - let me know.

Reply:
It does not harm the actual rendering. Due to the with of the column i can understand your pov. I have a very wide screen, so i personally prefer to have all columns full with. There is no rule preventing you from doing this, so let's continue :-)